test/RedditSourceTest.java

```diff
@@ -32,11 +34,16 @@ public void testAuthentication() {
     }
 
     @Test
-    public void testGetHotPosts() {
-        for(Submission s : redditSource.getHotPosts()) {
-            System.out.println(s);
-        }
-        assertEquals(100, redditSource.getHotPosts().size());
+    public void testGetTrendingPosts() {
+        List<Post> post = redditSource.getTrendingPosts("", 1, null);
```

Comment:
we should probably use some stubs here to remove dependencies on live data, but we can leave that for now.